Descriptions/Curation.md

```diff
@@ -0,0 +1,62 @@
+# Data Curation
+
+Here we declare the total data curation pipeline to provide consistent modeling development and prediction results. We use the same data curation rules for both training sets and prediction sets as they should be equally curated.
+For model development, we explore the experimental background from which the data was obtained, and for prediction, we strongly recommend using the applicability domain of developed models to ensure if they are applicable to studied biochemical problems.
+
+## Common pipeline for data from different sources
+
+The preliminary curation process guarantees that the data from all used sources is compatible with each other and standardized for further model development. We focus on the following steps:
+1. Removing data that is not associated with both structural data and property/activity value.
+2. Basically, structural information and a value or binary result are sufficient for modeling, but additional fields of data points are preferable to handle when present:
+- Source type - a patent, a manuscript, a commercial database with assays - these categories can provide valuable information as they usually study different chemical classes, and the quality and volume of data can vary significantly.
+- Value units - there is no hope to get accurate results with different units, fortunately they could be converted in most cases - like mM/L vs mg/L.
+- Endpoint type - different endpoints are difficult to mix for a single model development, properties are usually measured as a single endpoint, activities not  - Inh%, IC50, Ki are not even equal endpoints. Experimental conditions are not always present but play a crucial role, values obtained under different experimental conditions may interfere significantly, for example, solubility measured at 20 C or 35 C is different solubility. It is worth knowing and making different assumptions when a dataset is composed of data obtained in different experiments. Depending on the specific task, this field is just useful or crucial.
+    >In well-known databases, such fields are standardized:
+in PubChem, this information is provided in the following fields - “Panel Name” or “Panel Target”. In ChEMBL, this information is provided in the following fields - “Assay Description” or “Description”, “Assay Organism”, “Assay Cell Type”, “Target ChEMBL ID”, “Target Name”, “Target Organism”. We DO NOT extract this information from manuscripts. Examples:
+For antiviral activity, the “Assay description” may contain “Antiviral activity against HIV-1 in CEM cells” and “Inhibition of purified recombinant HIV-1 protease in a fluorogenic assay,” which means that the first experiment was in vivo and the second in vitro. Those two should never be mixed even if the studied endpoint is IC50 or Ki.
```

Comment:
Simplify. When a sentence takes 2 and more rows, it is HARD to read and understand. Especially when we have quite a big flow